### PR TITLE
adding unit tests for proxy get logs

### PR DESCRIPTION
**Pull request by s-radyuk**
EES-4262

### CODE REVIEW

File: pkg/proxy/proxy_test.go [hunk 1]
```diff
@@ -571,6 +571,120 @@ func TestProxyDelete(t *testing.T) {
...
+		//		},
+		//		Status: apisLoadTestV1.LoadTestStatus{
+		//			Phase:     apisLoadTestV1.LoadTestRunning,
+		//			Namespace: "aaa",
+		//		}},
+		//	http.StatusOK,
+		//	"{\"error\":\"some error\"}\n",
+		//	nil,
+		//	nil,
+		//},
```
Comment (aleksandrzhiliaev) [inline pkg/proxy/proxy_test.go:649]: debug leftover?

File: pkg/proxy/proxy_test.go [hunk 2]
```diff
@@ -571,6 +571,120 @@ func TestProxyDelete(t *testing.T) {
...
+
+			req = req.WithContext(ctx)
+
+			w := httptest.NewRecorder()
+			handler(w, req)
+
+			resp := w.Result()
+			respBody, _ := ioutil.ReadAll(resp.Body)
+
+			assert.Equal(t, tt.expectedCode, resp.StatusCode)
```
Comment (aleksandrzhiliaev) [inline pkg/proxy/proxy_test.go:681]:
```suggestion
			require.Equal(t, tt.expectedCode, resp.StatusCode)
```

File: pkg/proxy/proxy_test.go (hunk 1 above)
Comment (s-radyuk) [reply pkg/proxy/proxy_test.go]: piece of work in progress 😊

**Comment by aleksandrzhiliaev** [comment]
👍

**Pull request merged**